### PR TITLE
dev-python/pweave: Report generator, literate programming tool

### PR DESCRIPTION
**Pull request by AndrewAmmerlaan**
Package-Manager: Portage-2.3.84, Repoman-2.3.20
Signed-off-by: Andrew Ammerlaan <andrewammerlaan@riseup.net>

### CODE REVIEW

File: dev-python/pweave/pweave-0.30.3.ebuild [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+		https://github.com/mpastell/Pweave"
+SRC_URI="mirror://pypi/${MY_PN:0:1}/${MY_PN}/${MY_P}.tar.gz -> ${P}.tar.gz"
+
+IUSE="examples"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+PATCHES="${FILESDIR}/${P}-docs.patch
```
Comment (juippis) [inline dev-python/pweave/pweave-0.30.3.ebuild]: Array form here.

File: dev-python/markdown/markdown-3.1.1.ebuild [hunk 2]
```diff
@@ -3,7 +3,9 @@
 
 EAPI=7
 
-PYTHON_COMPAT=( python2_7 python3_{6,7} pypy3 )
+PYTHON_COMPAT=( python3_{6,7,8} pypy3 )
+
+DISTUTILS_USE_SETUPTOOLS=rdepend
```
Comment (juippis) [inline dev-python/markdown/markdown-3.1.1.ebuild]: It's not "broken" though is it? This change requires a revbump also.

File: dev-python/pweave/pweave-0.30.3.ebuild [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+MY_P="${MY_PN}-${PV}"
+
+DESCRIPTION="Scientific report generator and literate programming tool"
+HOMEPAGE="http://mpastell.com/pweave
+		https://github.com/mpastell/Pweave"
+SRC_URI="mirror://pypi/${MY_PN:0:1}/${MY_PN}/${MY_P}.tar.gz -> ${P}.tar.gz"
+
+IUSE="examples"
+
+LICENSE="MIT"
```
Comment (juippis) [inline dev-python/pweave/pweave-0.30.3.ebuild]: Also it doesn't look like a MIT license.

File: dev-python/markdown/markdown-3.1.1.ebuild (hunk 2 above)
Comment (AndrewAmmerlaan) [reply dev-python/markdown/markdown-3.1.1.ebuild]: Sorry, I wasn't very specific on what is broken

The docs flag is broken `esetup.py build_docs` is not a valid command and fails.

Also, when emerging it, it outputs a message telling me this `DISTUTILS_USE_SETUPTOOLS=rdepend` should be in the ebuild.

File: dev-python/pweave/pweave-0.30.3.ebuild [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+	dev-python/jupyter_client[${PYTHON_USEDEP}]
+	dev-python/markdown[${PYTHON_USEDEP}]
+	dev-python/nbconvert[${PYTHON_USEDEP}]
+	dev-python/nbformat[${PYTHON_USEDEP}]
+	dev-python/pygments[${PYTHON_USEDEP}]"
+
+S="${WORKDIR}/${MY_P}"
+
+distutils_enable_sphinx doc/source dev-python/sphinx_rtd_theme
+distutils_enable_tests pytest
```
Comment (juippis) [inline dev-python/pweave/pweave-0.30.3.ebuild]: Had to install matplotlib and scipy, which are documented by the way, to get tests to work.
https://github.com/mpastell/Pweave/blob/master/setup.py#L45

Add these and rest are ok with this PR!

Comment (AndrewAmmerlaan) [reply dev-python/pweave/pweave-0.30.3.ebuild]: Oops, I'm so sorry about that, I completely forgot to add the test dependencies. I just pushed the fix here.

**Pull request closed without merging**